string NAME=ascend

ObjectDrone d
ObjectLocation l0
ObjectLocation l1
